api/src/main/java/edu/cornell/mannlib/vitro/webapp/rdfservice/impl/RDFServiceImpl.java

```diff
@@ -317,47 +348,45 @@ protected Model[] separateStatementsWithBlankNodes(Model gm) {
     }
 
     protected Query createQuery(String queryString) throws RDFServiceException {
-        List<Syntax> syntaxes = Arrays.asList(
-                Syntax.defaultQuerySyntax, Syntax.syntaxSPARQL_11,
-                Syntax.syntaxSPARQL_10, Syntax.syntaxSPARQL, Syntax.syntaxARQ);
+        List<Syntax> syntaxes = Arrays.asList(Syntax.defaultQuerySyntax, Syntax.syntaxSPARQL_11, Syntax.syntaxSPARQL_10,
+                Syntax.syntaxSPARQL, Syntax.syntaxARQ);
         Query q = null;
         Iterator<Syntax> syntaxIt = syntaxes.iterator();
         while (q == null) {
             Syntax syntax = syntaxIt.next();
             try {
-               q = QueryFactory.create(queryString, syntax);
+                q = QueryFactory.create(queryString, syntax);
             } catch (QueryParseException e) {
-               if (!syntaxIt.hasNext()) {
-					throw new RDFServiceException("Failed to parse query \""
-							+ queryString + "\"", e);
-               }
+                if (!syntaxIt.hasNext()) {
+                    throw new RDFServiceException("Failed to parse query \"" + queryString + "\"", e);
+                }
             }
         }
         return q;
     }
 
-	@Override
-	public String toString() {
-		return ToString.simpleName(this) + "[" + ToString.hashHex(this) + "]";
-	}
+    @Override
+    public String toString() {
+        return ToString.simpleName(this) + "[" + ToString.hashHex(this) + "]";
+    }
 
     @Override
     public long countTriples(RDFNode subject, RDFNode predicate, RDFNode object) throws RDFServiceException {
         StringBuilder whereClause = new StringBuilder();
 
-        if ( subject != null ) {
+        if (subject != null) {
```

Comment:
You can switch to Objects.nonNull() everywhere but you don't have to if it's too much work, Its just my preference and is not necessarily better approach :smiley: